src/components/App/Pages/User/UserLayout/index.js

```diff
@@ -69,6 +70,8 @@ export const UserLayout = ({ children, location, showSideMenu }) => {
           <Heading as='h1' sx={{ margin: 0, fontSize: '6' }}>{portfolioUser.fullName}</Heading>
           <Box sx={{ py: '1rem' }}>
             <EditUserButton />
+            &nbsp;
```

Comment:
Didn't know if this was intentionally put here or not?

Reply:
yes